operators/pkg/controller/kibana/config/telemetry.go

```diff
@@ -5,25 +5,18 @@
 package config
 
 import (
+	"github.com/elastic/cloud-on-k8s/operators/pkg/info"
 	"github.com/ghodss/yaml"
 )
 
-const TelemetryFilename = "telemetry.yml"
-
-type Telemetry struct {
-	Eck Eck `json:"eck"`
-}
-
-type Eck struct {
-	Version string `json:"version"`
-}
+const telemetryFilename = "telemetry.yml"
 
+// getTelemetryYamlBytes returns the YAML bytes for the information on ECK.
 func getTelemetryYamlBytes() []byte {
-	t := Telemetry{
-		Eck{
-			"0.8.0",
-		},
-	}
-	bytes, _ := yaml.Marshal(t)
+	bytes, _ := yaml.Marshal(struct {
```

Comment:
Why the anonymous struct?

And why not return the possible `error`, even though it's theoretical? It seems all callers of this method can easily pass the error on?

Reply:
> Why the anonymous struct?
Why not? 
For real, the struct is only used to get the YAML.

> And why not return the possible `error`, even though it's theoretical? It seems all callers of this method can easily pass the error on?
I wanted to keep it simple. I believe this code can't return an error. But we can do as usual.

Reply:
You are right it should never fail atm, but some time down the line someone changes something with the data that is stored in this (e.g something in what is returned from `info.Get()` has a custom marshalling method that returns and err), and all of a sudden it can return an err, which is ignored. Business as usual is preferred I think :)